ui.frontend/src/main/webpack/components/_column.scss

```diff
@@ -0,0 +1,337 @@
+        {
+            background: $color-text-white !important;}
+
+            .cmp-column-title
+            {
+                color:$color-link;
+            }
+            ul li::after {
+                content: "";
+                position: absolute;
+                top: 50%;
+                left: 6px;
+                transform: translateY(-50%) rotate(45deg);
+                width: 6px;
+                height: 14px;
+                border-bottom: 3.5px solid $color-link;
+                border-right: 3.5px solid $color-link;
+            } 
+            .cmp-column-button
+            {
+                background: $color-link;
+                color: white;
+            }
+            .cmp-column-mobile-icon{
+                background: $color-link;
+
+            }
+            .cmp-column-mobile-number 
+            {
+                color: $color-link;   
+            } 
+    }
+  
+.cmp-column-red-color
+    {
+        .cmp-column-component-container
+        {
+            background: $color-text-white;}
+            .cmp-column-title
+            {
+                color:$color-text-red;
+            }
+            ul li::after {
+                content: "";
+                position: absolute;
+                top: 50%;
+                left: 6px;
+                transform: translateY(-50%) rotate(45deg);
+                width: 6px;
+                height: 14px;
+                border-bottom: 3.5px solid $color-text-red;
+                border-right: 3.5px solid $color-text-red;
+            } 
+            .cmp-column-button
+            {
+                background: $color-text-red;
+                color: white;
+            }
+            .cmp-column-mobile-icon{
+                background: $color-text-red;
+
+            }
+            .cmp-column-mobile-number 
+            {
+                color: $color-text-red;   
+            } 
+    }
+  
+.cmp-column-green-color
+    {
+        .cmp-column-component-container
+        {
+            background: $color-background;}
+            .cmp-column-title
+            {
+                color:$color-lightgreen;
+            }
+            ul li::after {
+                content: "";
+                position: absolute;
+                top: 50%;
+                left: 6px;
+                transform: translateY(-50%) rotate(45deg);
+                width: 6px;
+                height: 14px;
+                border-bottom: 3.5px solid $color-lightgreen;
+                border-right: 3.5px solid $color-lightgreen;
+            } 
+            .cmp-column-button
+            {
+                background: $color-lightgreen;
+                color: white;
+            }
+            .cmp-column-mobile-icon{
+                background: $color-lightgreen;
+
+            }
+            .cmp-column-mobile-number 
+            {
+                color: $color-lightgreen;   
+            } 
+    }
+   .cmp-column-blue-color{
+    
+   }
+  
+ 
```

Comment:
I don't see any media query in the file can you make sure we have responsive designs for the component.